composables/useProbeFilters.ts

```diff
@@ -12,12 +12,15 @@ export interface StatusOption {
 	outdatedOnly?: boolean;
 }
 
+export type SortOption = 'location' | 'tags' | 'name';
```

Comment:
Restrictive types like this make the code unnecessarily complicated and often don't really help. A simple string might be better here.